### PR TITLE
readme/instructor guide for workshop

### PR DESCRIPTION
**Pull request by cassws**
Hi all,

Here's a place for us to discuss the readme (which i've formatted here as a lightweight instructor guide). What would you like to see in this guide?

If you'd like to use GitHub's **Review Changes** functionality, you can click on the "Flies changed (1)" tab at the top of this Pull Request and click on the green Review Changes button (or click at the beginning of individual lines to add comments).

Thanks all!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# ethics-in-data-viz
\ No newline at end of file
+# Ethics in Data Visualization workshop
+
+## Introduction
+This hour-long workshop (undergrad and grad level) provides an introduction to ethics in data visualization. Please feel free to use this workshop, repurpose and remix this content, add suggestions to this project, etc.!
+
+This work is part of the IMLS-funded [Visualizing the Future](visualizingthefuture.github.io/) project.
```
Comment (amzoss) [inline README.md]: The official statement should be:

This project was made possible in part by the [Institute of Museum and Library Services](https://www.imls.gov/), RE-73-18-0059-18.

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
-# ethics-in-data-viz
\ No newline at end of file
+# Ethics in Data Visualization workshop
+
+## Introduction
+This hour-long workshop (undergrad and grad level) provides an introduction to ethics in data visualization. Please feel free to use this workshop, repurpose and remix this content, add suggestions to this project, etc.!
```
Comment (amzoss) [inline README.md]: Maybe remove reference to hour-long, since there are a variety of ways to run the workshop.

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+## The goals for this workshop
+
+By the end of this sessions, learners will be able to:
+
+* List the five steps in the data visualization creation process: select topic, problem, or question; acquire data; clean data; analyze data; create the visualization
+* Understand how decisions in the steps up to data visualization can affect the final visualization 
+* Identify common sources of bias in the data visualization process
+* Practice ethics-centered data visualization design
+
+
```
Comment (amzoss) [inline README.md:39]: Add section that specifies the suggested times for different parts of the workshop:

Lecture: 15-20 min lecturing, including individual/small group reflection questions
1st Activity - Mapping Census (ACS) Data: 10-20 min full group, 15-30 min small group
2nd Activity - Redesigning Economist Charts: 15-30 full group, 30-50 min small group

File: README.md [hunk 4]
```diff
@@ -1 +1,35 @@
...
+# Ethics in Data Visualization workshop
+
+## Introduction
+This hour-long workshop (undergrad and grad level) provides an introduction to ethics in data visualization. Please feel free to use this workshop, repurpose and remix this content, add suggestions to this project, etc.!
+
+This work is part of the IMLS-funded [Visualizing the Future](visualizingthefuture.github.io/) project.
+
+## How to teach this course
+
+We provide two starting points for teaching Ethics in Data Visualization:"
```
Comment (amzoss) [inline README.md]: Extra " at the end of line?

**Review by amzoss — CHANGES_REQUESTED** [review]
I'm happy to help with any/all of these changes after PR merge, too! Either way works.

File: README.md [hunk 5]
```diff
@@ -1 +1,35 @@
...
+
+### 1. A single, self-contained instructor slideshow 
+
+This slideshow includes everything you'll need to lead the workshop. In addition to learner-facing slides, there are also hidden instructor slides, speaker notes, etc.
+
+
+### 2. Directories with extra materials
+
+We've included a variety of materials to make in-person instruction easier, and also to help you excerpt some of the activities and ideas here for use in another course. These include:
```
Comment (grynoch) [inline README.md:19]: These materials can also help with online instruction

**Comment by cassws** [comment]
Thanks @amzoss @grynoch ! Incorporated your suggestions into this PR.

**Review by amzoss — APPROVED** [review]

**Pull request merged**